src/components/ItemFilter.vue

```diff
@@ -40,26 +40,62 @@
         />
       </b-form-group>
 
+      <div class="additionalFilters">
+        <b-form-group label="Select additional field filters" label-for="availableFields" description="Fields advertised by the /queryables endpoint">
+          <b-dropdown size="sm" text="Add field filter" block variant="primary" class="m-2" menu-class="w-100">
+            <b-dropdown-item v-for="option in fieldFilterOptions" :key="option.text" @click="additionalFieldSelected(option)">{{ option.text }}</b-dropdown-item>
```

Comment:
I've tested this against MS PC (prod and staging), but the lift of filters is empty for me?

Reply:
I haven't done the global search side of things yet, I've been focussed on doing things at the collection level and getting my head around the app architecture, but I'll look into the global search side now.